### DISCUSSION TITLE
Support Smart Connections by voting

### DISCUSSION THREAD

**Discussion by quelsemme** [brianpetro/obsidian-smart-connections#406] [General]
If you're on the Obsidian discord, don't forget to vote for Smart Connections under Best New Plugin. I voted this morning. 

It's worth joining the discord anyway, for other Obsidian news and troubleshooting, but let's support @brianpetro while we're at it.

https://discord.com/channels/686053708261228577/702717892533157999/1193993954346139729

**Comment by quelsemme** [comment]
Apologies for not screenshotting, Brian. I did it before I saw the post in the plugin.

**Reply by brianpetro** [reply]
@quelsemme just send me an email and I'll set you up.

🌴
